Source/Web/app/information.js

```diff
@@ -616,6 +686,12 @@ function dispWork(id) {
         "<div class='form-group'><label class='control-label col-sm-2' for='text'>Worker Surname:</label>" +
         "<div class='col-sm-9'><input type='text' class='form-control' id='workSName'></div> </div>" +
         "" +
+	"<div class='form-group'><label class='control-label col-sm-2' for='text'>Identity Number:</label>" +
+	"<div class='col-sm-9'><input type='text' class='form-control' id='workID'></div> </div>" +
+        "" +
+	"<div class='form-group'><label class='control-label col-sm-2' for='text'>Phone Number:</label>" +
+	"<div class='col-sm-9'><input type='text' class='form-control' id='workContactNo'></div> </div>" +
+        "" +
         "<div class='form-group'><label class='control-label col-sm-2' for='sel1'>Assigned Orchard:</label>" +
```

Comment:
Assigned Orchards don't show up. For example Bruce Wayne is assigned to 3 orchards but they don't show up here.

Reply:
Ow the working for collection isn't getting an update. Will fix that now